components/simplified-grid/simplified-grid.css

```diff
@@ -0,0 +1,28 @@
+@import url("/docs/main.css");
+
+.grid_container {
+    display: grid;
+    background-color: var(--light-grey-color);
+    width: fit-content;
+    height: fit-content;
+}
+
+.grid_item {
+    padding: 0.5em;
+} 
+
+.grid_img {
+    height: 100%;
+}
+
+.grid_two_columns {
+    grid-template-columns: 1fr 1fr;
+    gap: 10px;
+    padding: 1rem;
+}
+
+.grid_three_columns {
+    grid-template-columns: 1fr 1fr 1fr;
```

Comment:
gsuggestion: You can use shorthand property like grid-template-columns: repeat(3,1fr)

Reply:
Yea, I'll do that. Thanks, Gopal.